api/rest/v2/node.md

```diff
@@ -56,26 +56,27 @@ X-API-Key: {your_api_key}
 
 JSON Data:
 {
-  "nodes": [
-      "{value_1}",
-      "{value_2}",
-      ...
-    ],
-  "property": "{property_expression}"
+"nodes": [
```

Comment:
Can you revert this? I think this would be from the auto format from vscode. You may want to turn that off.